torchmultimodal/diffusion_labs/models/adm_unet/adm.py

```diff
@@ -202,10 +106,40 @@ def __init__(
         use_attention_for_layer: List[bool],
         dim_res_cond: int,
         dim_attn_cond: Optional[int] = None,
+        embed_dim: Optional[int] = None,
         in_channels: int = 3,
         out_channels: int = 3,
+        time_embed_dim: Optional[int] = None,
+        timestep_encoder: Optional[nn.Module] = None,
```

Comment:
I'm not sure if it makes sense to make these three encoders/projectors a parameter. It makes the init much more messy. I'd be in favor of removing that option.

Reply:
After our offline discussion, I plan to leave these in for this version to unblock other changes. But let's remove in a follow-up